### DISCUSSION TITLE
Please release a mini version with low VRAM!

### DISCUSSION THREAD

**Discussion by dream80** [xai-org/grok-1#103] [General]
The current model cannot be used by ordinary people at all. We need a model with smaller parameters and good capabilities.  A good model is one that everyone can play with.

Anyone have the same idea?

**Comment by bluevisor** [comment]
I'm sure there are people who know what they are doing already working on this.

**Reply by StavrosD** [reply]
This is a HUDGE model, about 300B parameters. With 8bit quantization you need about 300GB VRAM. 
Even if you quantize it to 1bit (worst quality) you need 100GB VRAM, you need 5 GPUs with 24GB VRAM each. 
If we are lucky we will see a quantized GGUF version. GGUF supports loading to RAM and offloading some layers to the VRAM.

**Reply by AdaptiveStep** [reply]
You could run this as an entire game engine, and let just its context size be responsible for the entire game state lol.

**Reply by nukedsystem** [reply]
Is it a model for millionaires or billionaires? Is it an Elite model?
